### PR TITLE
Development complete

### PR DESCRIPTION
**Pull request by darshanjayadev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <li>About</li>
+            <li>Privacy</li>
+            <li>Terms of use</li>
+            <li>community</li>
+          </ul>
+        </div>
+      </div>
+        
+    </footer>
+    </div>
```
Comment (inhaq) [inline index.html]: this `div` should be within `footer`

**Review by inhaq — CHANGES_REQUESTED** [review]
### Good job 🏅
Everything is good and working as it should. 
Only concern is that, framework is used because of not to write our own custom SCSS. In this project, you're only declaring grid and card for framework and rest of the stuff you're doing in custom SCSS. It would be great if you cut down most of your custom SCSS and put it in framework SCSS. Let me DM on slack if you've any queries. Once fixed, kindly resubmit.

**Review by inhaq — APPROVED** [review]
### 👍🏻for this project
Congrats on completing this project 👏🏻go ahead and submit a project completion form.
I would recommend adding further functionalities to the framework as well. 
Good luck with the next project 🎉

**Comment by darshanjayadev** [comment]
@inhaq Our framework now contains paddings, margins, buttons, jumbotron, container, grids

**Comment by inhaq** [comment]
@daashandayo now instead of writing custom SCSS for your demo site, use your framework classes and remove custom SCSS from the framework.

**Comment by darshanjayadev** [comment]
@inhaq Yeah we removed Custom SCSS and included mixins for margin, padding and jumbotron instead.

**Pull request merged**